packages/controller/src/main.ts

```diff
@@ -2595,6 +2602,25 @@ async function processMessage(msg: ioBroker.SendableMessage): Promise<null | voi
                             logger.warn(`${hostLogPrefix} Repository object could not be updated: ${e.message}`);
                         }
                     }
+
+                    try {
+                        const upgradedAdapters = await autoUpgradeManager.upgradeAdapters();
```

Comment:
we need a "blocker here" to prevent that this is done and checked too often and might obverlap in runs!

Reply:
yes, we have the blocker for too many repo updates, but yes it can still overlap. we should add a guard